test/hoverable.test.js

```diff
@@ -0,0 +1,50 @@
+/*global describe, afterEach, it*/
+'use strict';
+
+var expect = require('expect.js');
+
+var oHoverable = require('./../main.js');
+
+describe('o-hoverable', function() {
+
+	var testHoverable;
+
+	afterEach(function() {
+		testHoverable.destroy();
+	});
+
+	it('should initialise the module', function() {
+		expect(window.document.body.hasAttribute('data-o-hoverable--js')).to.be(false);
+		testHoverable = new oHoverable();
+		expect(window.document.body.hasAttribute('data-o-hoverable--js')).to.be(true);
+	});
+
+	it('should destroy the module', function() {
```

Comment:
Typically I'd describe _what_ should destroy the module in the test description, something along the lines of:

`it('should destroy the module when #destroy() is called', function() {})`;

Alternatively - I prefer the more verbose Mocha BDD style, but this is a matter of preference, I just feel it's a better way of organising the tests and gives them the nice property of becoming fairly good documentation:

```JS
describe("module", function() {
  describe("#method(arg1, arg2)", function() {
     it("should do x", function() {});
     it("should do y", function() {});
  });

  describe("#methodA()", function() {
     it("should do z", function() {});
  });
});
```